src/helpers/collectCss.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const minifyCss = require('./minifyCss');
+const {
+  appendChild,
+  insertText,
+  createElement,
+  nextNode,
+  firstChildByTag,
+  remove,
+} = require('@ampproject/toolbox-optimizer').NodeUtils;
+
+class CollectCss {
+  constructor(config) {
+    this.minifyCss = !!config.minifyCss || true;
```

Comment:
this only sets to true

Reply:
unless im misunderstanding its purpose, this should be something like 
```js
'minifyCss' in config ? !!config.minifyCss : true
```

Reply:
🤦Fixed. Also added test...